tests/providers/discord/notifications/test_discord.py

```diff
@@ -0,0 +1,58 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from airflow.models import Connection
+from airflow.providers.discord.notifications.discord import DiscordNotifier
+from airflow.utils import db
+
+
+@pytest.fixture(autouse=True)
+def setup():
+    db.merge_conn(
+        Connection(
+            conn_id="my_discord_conn_id",
+            conn_type="HTTP",
```

Comment:
Nit: I think i's better to test with `discord` connection type
https://github.com/apache/airflow/blob/3193857376bc2c8cd2eb133017be1e8cbcaa8405/airflow/providers/discord/hooks/discord_webhook.py#L53-L53